.github/pull_request_template.md

```diff
@@ -6,12 +6,13 @@ Please write a few sentences describing the overall goals of the pull request's
 
 ## PR checklist
 
-Please tick as appropriate and edit the reasons (e.g.: "backport: not needed because this is a new feature")
+Please remove add/remove strikethrough ~~ as appropriate, and add any relevant link/s to the end of the line.
 
-- [ ] **changelog** provided, or not required
-- [ ] **3.6 backport** done, or not required
-- [ ] **2.28 backport** done, or not required
-- [ ] **tests** provided, or not required
+- [ ] **changelog** provided, ~~not required~~, ~~covered~~ . Link --> #XYZ
+- [ ] **dev PR** provided, ~~not required~~ Link --> #XYZ
```

Comment:
I would expect this to be filled with `not required` because this is the backport as discussed [above ](https://github.com/Mbed-TLS/mbedtls/pull/9080#discussion_r1595435482).

And again if someone does a provided and self references, that should also work.

But I would refrain from changing the keywords for each entry, since that comes with a significant cost of maintaining the parsing script. 


code like that won't work if each of the lines is unique

~~~~~~~
checkbox_rex = re.complile(r'....')

... find anchors for start/end of checkboxes

for line in text:
  checkbox_rex .search(line)
~~~~~~~

Reply:
How about something like this: `provided: #PR, ~not required because:~`